services/workflows-service/src/alert/alert.service.intg.test.ts

```diff
@@ -76,7 +77,6 @@ describe('AlertService', () => {
       baseTransactionFactory = transactionFactory
         .project(project)
         .paymentMethod(PaymentMethod.credit_card)
-        .withBusinessOriginator()
```

Comment:
I removed this from the base factory because I don't need it in the new tests I added, so I had to add it to the other tests.